src/dojos/cd-find-dojo.vue

```diff
@@ -107,6 +106,7 @@
           .then((coords) => {
             this.coordinates = coords;
             this.getDojosByLatLong();
+            this.$router.push({ path: `/?q=${this.searchCriteria}` });
```

Comment:
Should also probably actually move this up to the top of the `searchDojosByAddress` function. That way, we change the URL immediately.

The one for detect location is fine since it'll wait until the user accepts to share their location before it'll change the URL, but here, we should be able to hit refresh immediately after we search and re-execute the same search.